sigs.yaml

```diff
@@ -3326,6 +3326,48 @@ workinggroups:
     liaison:
       github: pohly
       name: Patrick Ohly
+- dir: wg-device-management
+  name: Device Management
+  mission_statement: >
+    Enable simple and efficient configuration, sharing, and allocation of accelerators
+    and other specialized devices.
+
+    [Additional context](https://groups.google.com/a/kubernetes.io/g/dev/c/YWXGXe07A5w/m/OqLvdQ47BQAJ)
+
+  charter_link: charter.md
+  stakeholder_sigs:
+  - Architecture
+  - Autoscaling
+  - Network
+  - Node
+  - Scheduling
```

Comment:
Is there a case here for adding Multicluster as a stakeholder SIG? I'm thinking in particular of use cases that rely upon hw acceleration at massive scale (spanning many clusters). Having a common standard to aid w/ scheduling and hw capability discovery across many clusters would greatly simplify this problem space.

cc @JeremyOT @pmorie @lauralorenz

Reply:
We can always amend this document if it becomes necessary. Pretty sure we'll end up with more than the sigs listed here as participants.